Resources/Prototypes/Entities/Buildings/airlock_types.yml

```diff
@@ -127,6 +112,103 @@
   components:
   - type: Sprite
     sprite: Buildings/Doors/airlock_security_glass.rsi
-
   - type: Icon
     sprite: Buildings/Doors/airlock_security_glass.rsi
+
+- type: entity
+  parent: Airlock
+  id: AirlockMaint
+  name: Maintenance Hatch
+  components:
+  - type: Sprite
+    sprite: Buildings/Doors/airlock_maint.rsi
+  - type: Icon
+    sprite: Buildings/Doors/airlock_maint.rsi
+
+- type: entity
+  parent: Airlock
+  id: AirlockMaintCargo
+  name: Maintenance Hatch
+  suffix: Cargo
+  components:
+  - type: Sprite
+    sprite: Buildings/Doors/airlock_maint_cargo.rsi
+  - type: Icon
+    sprite: Buildings/Doors/airlock_maint_cargo.rsi
+
+- type: entity
+  parent: Airlock
+  id: AirlockMaintCommand
+  name: Maintenance Hatch
+  suffix: Command
+  components:
+  - type: Sprite
+    sprite: Buildings/Doors/airlock_maint_command.rsi
+  - type: Icon
+    sprite: Buildings/Doors/airlock_maint_command.rsi
+
+- type: entity
+  parent: Airlock
+  id: AirlockMaintCommon
+  name: Maintenance Hatch
+  suffix: Common
```

Comment:
I dont thing is a good idea to use suffix in this way, they should be used to show information that the players doesnt need but that is useful for mappers/admins/etc. Players need the feedback of knowing what access they need to enter a door and if you use suffix they wont be able to see it.

Reply:
Why not. I copied the behavior from CEV-Eris, where the maintenance doors are not named IIRC.

Reply:
This is fine.